src/pages/blog/kata-containers-3-5-0-release-overview.md

```diff
@@ -0,0 +1,42 @@
+---
+templateKey: blog-post
+title: Kata Containers 3.5.0 Release Update
+author: Ildiko Vancsa
+date: 2024-06-19T01:32:05.627Z
+category:
+  - value: category-edic1zzR0
+    label: News & Announcements
+---
+
+The Kata Containers community has been making improvements to the project’s release process to be able deliver new versions more frequently, aiming for a monthly minor release cadence.
+
+The latest [3.5.0](https://github.com/kata-containers/kata-containers/releases/tag/3.5.0) release focuses on improving the project's stability, security, and compatibility through various bug fixes, updates, and new features. Key updates include enhanced support for NVIDIA GPUs, improvements in the runtime and agent components, and enhancing the container creation process and storage handling. Additionally, this version updates critical dependencies like ‘golang’ and ‘libseccomp’ for enhanced security.
+
+# Key Themes for the New Release
+
+1. Bug Fixes
+
+The new release brings increased stability and better performance. Some of the most affected areas include:
+- ‘runtime-rs’: Fixed cleanup after VM start failure with Dragonball and the contribution of ‘RTC struct’.
+- ‘build’: Improved build messages and fixed tarball build in Docker, to ensure a smoother development experience.
+
+2. Security Enhancements
+
+The kata-agent now includes the latest ‘libseccomp’ v2.5.5 to improve security. Please note that this library is licensed under [GNU LGPL-2.1](https://www.gnu.org/licenses/old-licenses/lgpl-2.1.html). The version used by Kata Containers is not modified from the upstream version, and you can find the complete source code for the library attached for full compliance.
+
+In addition, ‘rootfs’ is now built with≈ with ‘AGENT_POLICY=yes’ by default for increased security and stability.
```

Comment:
I think it's worth mentioning related to this that in 3.5.0 we changed the policy implementation from the golang based OPA binary to the regorus crate, which should give improved performances and a smaller rootfs and memory performance of the guest.

Reply:
This might be in new features section though.

Reply:
I added it under the new features section.